src/components/Home/Slider.jsx

```diff
@@ -1,92 +0,0 @@
-import { Box } from "@chakra-ui/react";
-
-// Import Swiper React components
-import { Swiper, SwiperSlide } from "swiper/react";
-
-// Import Swiper styles
-import "swiper/css";
-
-import "swiper/css/pagination";
-import "swiper/css/navigation";
-
-import "../../styles/slider.css";
-
-// import required modules
-import { Pagination, Navigation } from "swiper";
-import { SliderCard } from "./SliderCard";
-
-export const Slider = () => {
-  return (
-    <Box className="swiper-container" bg={"#d5303a"}>
-      <Swiper
-        // pagination={{
-        //   type: "fraction",
-        // }}
-        navigation={true}
-        modules={[Pagination, Navigation]}
-        className="mySwiper"
-      >
-        <SwiperSlide>
```

Comment:
Make one reusable component and try to reuse it here, rather than copy pasting it several times.